crates/swc_ecma_transforms_typescript/src/strip.rs

```diff
@@ -1726,27 +2070,6 @@ where
 
     fn visit_mut_if_stmt(&mut self, s: &mut IfStmt) {
         s.visit_mut_children_with(self);
-        let span = s.span;
-
-        s.cons.map_with_mut(|cons| match *cons {
-            Stmt::Decl(Decl::TsEnum(e)) => {
-                let mut stmts = vec![];
-                self.handle_enum(e, &mut stmts);
-                Box::new(Stmt::Block(BlockStmt { span, stmts }))
-            }
-            _ => cons,
-        });
-
-        s.alt.map_with_mut(|alt| {
-            alt.map(|s| match *s {
-                Stmt::Decl(Decl::TsEnum(e)) => {
-                    let mut stmts = vec![];
-                    self.handle_enum(e, &mut stmts);
-                    Box::new(Stmt::Block(BlockStmt { span, stmts }))
-                }
-                _ => s,
-            })
-        })
```

Comment:
This code was removed because I don't ever think these would have happened because there is an `s.visit_mut_children_with(self)` as the first statement in this method. Also, I'm not sure what this code was trying to do.